Model/Service/Merchandise/RequestParamsService.php

```diff
@@ -224,6 +225,9 @@ private function getCurrentCategoryString(Store $store)
     {
         /** @noinspection PhpDeprecationInspection */
         $category = $this->registry->registry('current_category'); //@phan-suppress-current-line PhanDeprecatedFunction
+        if (!$category instanceof Category) {
+            return null;
```

Comment:
Would it be better to throw an exception? Otherwise it will make a CM graphql query with category as null

Reply:
Good point, now it will bubble up and use the fallback sorting. Thanks man :)